dataloader/dataloader_impl.hpp

```diff
@@ -61,6 +61,41 @@ template<
               datasetMap[dataset].startTestingInputFeatures,
               datasetMap[dataset].endTestingInputFeatures);
     }
+    else if (datasetMap[dataset].datasetType == "image-detection")
+    {
+      std::vector<std::string> augmentations = augmentation;
+
+      // If user doesn't set size for images, set size of images to {64, 64}.
+      if (augmentations.size() == 0)
+        augmentations.push_back("resize = {64, 64}");
```

Comment:
Do you think we should print a warning here?

Reply:
Agreed, I'll add a warning here. Thanks.

Reply:
It seems you missed this comment :D

Reply:
Hey @KimSangYeon-DGU, added the warning before this line that's why it didn't show up here. Here is the [link](https://github.com/mlpack/models/pull/18/files#diff-d47cdd2f19276dd4d53c5cf0f5c3833fR71)

Reply:
Thanks for letting me know!